### PR TITLE
Overhaul contributing documentation for Plone 6

### LINKED ISSUE #357: Update contribution docs
Currently, the docs on how to contribute still reference the makefile, which no longer exists. These should be updated so that others can easily contribute. I believe the suggested way to get a dev environment these days is to use the coredev buildout, is that correct?

### LINKED ISSUE #473: Replace Netlify preview build command with 'tox -e docs' and remove Makefile
Do you want to keep the Makefile or use tox?

_Originally posted by @stevepiercy in https://github.com/plone/plone.api/pull/469#discussion_r844825454_

'tox -e docs' needs tox to be installed.

### PR DESCRIPTION
**Pull request by stevepiercy**
Now that we have Plone core contributing documentation, I've simplified and modernized the contributing documentation.

See https://github.com/plone/documentation/issues/1278.

- Closes #473
- Closes #357

After merging, then merge https://github.com/plone/documentation/pull/1684.

### CODE REVIEW

**Comment by stevepiercy** [comment]
@jenkins-plone-org please run jobs

**Comment by stevepiercy** [comment]
I have no clue as to why Jenkins fails for Plone 6.0/Python 3.11 but not for Plone 6.0/Python 3.8. Help!

File: docs/contribute.md [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+Alternatively, you can automatically reload changes to the documentation as you edit it in a web browser.
+
+```shell
+tox -e livehtml
+```
+
+The [`plone.api` documentation](https://6.docs.plone.org/plone.api) is automatically generated from the documentation source files when its submodule is updated in the [main Plone `documentation` repository](https://github.com/plone/documentation/).
+
+
+### Add a function to an existing module
```
Comment (ksuess) [inline docs/contribute.md:134]: This is a 2nd level heading, not a subheading under "Documentation"

Comment (ksuess) [inline docs/contribute.md:134]:
```suggestion
### Add a method to an existing module
```

**Review by ksuess — CHANGES_REQUESTED** [review]
I appreciate your overhaul. Just minor suggestions to content that was already there before this PR.

File: docs/contribute.md [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+## Continuous integration
+
+`plone.api` uses GitHub workflows for continuous integration.
+On every push to the `master` branch, GitHub runs its workflows for all tests and code quality checks.
+GitHub workflows are configured in the directory `.github/workflows` at the root of this package.
+
+
+## Documentation
+
+For every feature change or addition to `plone.api`, you should add documentation of it.
```
Comment (jensens) [inline docs/contribute.md:112]: `you should add documentation`-> it's a `must` here, isn't  it?

File: docs/contribute.md (hunk 1 above)
Comment (jensens) [reply docs/contribute.md:134]: Well, the whole plone.api is build with functions (not classes+methods), so it is explained below. So IMO functions is fine.

**Review by jensens — COMMENTED** [review]
thanks! Minor things see comments.

File: docs/contribute.md (hunk 1 above)
Comment (stevepiercy) [reply docs/contribute.md]: Fixed in b4e53e8c081d08906b3e5092017d72a424f2c991.

**Comment by stevepiercy** [comment]
@jenkins-plone-org please run jobs

**Comment by stevepiercy** [comment]
Changes made, and I excluded GNU make as a requirement here since this project uses tox instead.

It would be nice to use one of make or tox for all of Plone, don't you think?

**Comment by ksuess** [comment]
> It would be nice to use one of make or tox for all of Plone, don't you think?

Yes, a switch to `make` would be consistent. Definitly.

File: docs/contribute.md (hunk 1 above)
Comment (ksuess) [reply docs/contribute.md]: The implementation of a feature is of course a function, not a classes method. That's why I wrote this header like this back then.
It's just that I realised that in the rest of the documentation it is often spoken of 'API methods'.
But I am fine with keeping the header like it is.

**Review by ksuess — APPROVED** [review]

**Comment by stevepiercy** [comment]
> > It would be nice to use one of make or tox for all of Plone, don't you think?
> 
> Yes, a switch to `make` would be consistent. Definitly.

See https://community.plone.org/t/tools-for-development-gnu-make-tox-or/19708

**Pull request merged**